contracts/ERC721A.sol

```diff
@@ -504,6 +504,19 @@ contract ERC721A is IERC721A {
         _afterTokenTransfers(address(0), to, startTokenId, quantity);
     }
 
+    /**
+     * @dev Zeroes out _tokenApprovals[tokenId]
+     */
+    function deleteTokenApproval(uint256 tokenId) private {
+        mapping(uint256 => address) storage tokenApprovalPtr = _tokenApprovals;
+        assembly {
+            mstore(0, tokenId)
+            mstore(32, tokenApprovalPtr.slot)
```

Comment:
Nit: Some people (e.g. me) prefer hexadecimal (e.g. '0x00', '0x20', '0x40') for memory addresses. The Yul docs use hex for the memory addresses.